connectivity/drivers/lora/COMPONENT_SX1276/SX1276_LoRaRadio.cpp

```diff
@@ -1263,13 +1264,24 @@ void SX1276_LoRaRadio::read_fifo(uint8_t *buffer, uint8_t size)
 void SX1276_LoRaRadio::set_operation_mode(uint8_t mode)
 {
     if (mode == RF_OPMODE_SLEEP) {
+        write_to_register(REG_OPMODE, (read_register(REG_OPMODE) & RF_OPMODE_MASK) | mode);
         set_low_power_mode();
+        /* FIXME taken from set_modem, it reduces the power consumption
+         *from 300µA to 1.5µA */
+        write_to_register(REG_DIOMAPPING1, 0x00); // sets DIO0-DI03 in default mode
+        write_to_register(REG_DIOMAPPING2, 0x30); // bits 4-5 are turned on i.e.,
+        if (tcxo != NC) {
```

Comment:
This doesn't build...

Reply:
sorry not my original fix, will check that

Reply:
Should be better now, let's see